compiler/Main.hs

```diff
@@ -116,6 +118,12 @@ csCodegen options@Cs {..} = do
                         ]
 csCodegen _ = error "csCodegen: impossible happened."
 
+anySeriveInheritance :: [Declaration] -> Bool
+anySeriveInheritance = getAny . F.foldMap serviceWithBase
```

Comment:
Spelling: "anyServiceInheritance" (weird, I thought I'd already made this comment).